modules/ROOT/pages/auradb/managing-databases/database-actions.adoc

```diff
@@ -115,42 +120,75 @@ You cannot clone from a Neo4j version 5 instance to a Neo4j version 4 instance.
 === Clone to a new AuraDB instance
 
 . Select the ellipsis (*...*) button on the instance you want to clone.
-. Select *Clone To New* and then *AuraDB* from the resulting menus.
-. Set your desired settings for the new instance. For more information on AuraDB instance creation settings, see xref:auradb/getting-started/create-database.adoc[].
-. Tick the *I understand* checkbox and select *Clone Instance*.
+. Select *Clone To New* and then *AuraDB* from the contextual menu.
```

Comment:
It does not actually say "AuraDB" on the menu, it will either say "AuraDB Professional" or "AuraDB Enterprise" depending on the Tenant type. Not sure if this is something we want to be more specific at, maybe this is just fine.

![image](https://github.com/neo4j/docs-aura/assets/124151747/c4208a76-7b29-4b15-9698-0966f744fa5f)

Reply:
No, I agree, this should be pointed out! Thanks for spotting it!